.gitignore

```diff
@@ -1 +1,3 @@
 /node_modules/
+dist
+yarn.lock
```

Comment:
> - I suggest to exclude yarn.lock from git repo in 89ccf2a, it causes troubles when you use different registry. I really wonder why yarn devs decide to store registry URL inside the lock file...

It's important to add lockfiles to version control, though this means a project can only use a single package manager. [Here is a nice article](https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/) that goes in-depth about the reasons.  :slightly_smiling_face:

```suggestion
```

Reply:
I totally understand that lockfiles should be version controlled, and I'm actually committing my `package-lock.json`s or `yarn.lock`s in private projects.
The only reason I'm excluding `yarn.lock` here is they stores registry URL in the lock file. Imagine someone use a mirror registry which is extraordinary slow to access outside specified areas... That would slow down project sync, CI build, etc.
Anyway, I've included the lockfile and re-installed everything using `registry.npmjs.org`, it shouldn't be a problem now :)

Reply:
Oh, I see. Is the yarn registry slow for you?

Reply:
Not too slow, but the registry I'm using might be slow for most people, so I'll either exclude yarn.lock from git or re-install everything using npmjs.org registry when doing PRs ;)

Reply:
Can you use `https://registry.yarnpkg.com`? It's what yarn uses by default and apparently it's faster than the NPM registry.
It doesn't matter much since it's just a mirror but I have (and most contributors will have) the default yarn configuration, and I prefer avoiding having a mix of multiple registries for no reason. 🙂

Reply:
done